edgelet/build/windows/openssl.ps1

```diff
@@ -29,12 +35,14 @@ function Get-OpenSSL
     {
         New-Item -Type Directory "$env:HOMEDRIVE\vcpkg\Downloads" | Out-Null
     }
-    $strawberryPerlUri = "https://edgebuild.blob.core.windows.net/strawberry-perl/strawberry-perl-5.24.1.1-32bit-portable.zip"
-    $strawberryPerlPath = "$env:HOMEDRIVE\vcpkg\Downloads\strawberry-perl-5.24.1.1-32bit-portable.zip"
-    Invoke-WebRequest -Uri $strawberryPerlUri -OutFile $strawberryPerlPath
+    
+    # strawberry perl is now being downloaded as part vcpkg setup, no need to manually set up
```

Comment:
We download it from our storage account because it has happened in the past where its download server went offline and broke our build. Please put this code back.